pytorch_lightning/callbacks/pruning.py

```diff
@@ -27,10 +27,10 @@
 import pytorch_lightning as pl
 from pytorch_lightning.callbacks.base import Callback
 from pytorch_lightning.core.lightning import LightningModule
-from pytorch_lightning.utilities import _PYTORCH_PRUNE_AVAILABLE
+from pytorch_lightning.utilities import _TORCH_PRUNE_AVAILABLE
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
-if _PYTORCH_PRUNE_AVAILABLE:
+if _TORCH_PRUNE_AVAILABLE:
```

Comment:
Undoing these as it will generate conflicts with #5825 

```suggestion
from pytorch_lightning.utilities import _PYTORCH_PRUNE_AVAILABLE
from pytorch_lightning.utilities.exceptions import MisconfigurationException

if _PYTORCH_PRUNE_AVAILABLE:
```